blockdevice/util.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Very good thought on utils.
Lets try with `util/blockdevice/` as the package structure. Rest all remains same.

Reply:
done